src/helpers/buildTransactionsSectionsSelector.tsx

```diff
@@ -87,7 +101,9 @@ const buildTransactionsSections = (
     requestsToApprove = [
       {
         data: requests,
-        renderItem: renderItemElement(RequestCoinRow),
+        renderItem: ({ item }: any) => (
```

Comment:
Same about any

Reply:
Discussed in DMs. We will leave it as is for now.